config/main.py

```diff
@@ -288,13 +288,10 @@ def _abort_if_false(ctx, param, value):
 
 def _stop_services():
```

Comment:
In the start and stop services section it looks like these were removed.  Not sure why but I just updated the file from the main repository and saw this had changed.   Can you please check this part?

Reply:
You need to rebase (or merge) the latest changes into your branch. These changes shouldn't appear in your PR.